AGENT_CHANGELOG.md

```diff
@@ -21,11 +65,13 @@
 * etcd [1.7.0](https://github.com/DataDog/integrations-core/blob/master/etcd/CHANGELOG.md)
 * Exchange Server [1.3.0](https://github.com/DataDog/integrations-core/blob/master/exchange_server/CHANGELOG.md)
 * Gitlab Runner [2.1.0](https://github.com/DataDog/integrations-core/blob/master/gitlab_runner/CHANGELOG.md)
+* go_metro [1.1.0](https://github.com/DataDog/integrations-core/blob/master/go_metro/CHANGELOG.md)
```

Comment:
Why is the name different than L263?

Reply:
I think it's because at some point we switched from `folder_name` to `datadog-package-name` in the agent-reqs file and that messes things up for go-metro which has a `-` in its name. But it's not even supposed to be there in any case, i'll remove